site/enqueue.php

```diff
@@ -6,16 +6,23 @@ function zume_training_magic_url_base_allowed_js( $allowed_js = [] ) {
     $allowed_js[] = 'jquery';
     $allowed_js[] = 'jquery-core';
     $allowed_js[] = 'jquery-migrate';
-    $allowed_js[] = 'lodash';
-    $allowed_js[] = 'lodash-core';
     $allowed_js[] = 'moment';
```

Comment:
@squigglybob,
Just fyi moment is old an no longer maintained. We're slowly removing it from D.T

Reply:
👍🏻 great thanks. We weren't really using it, so  i've pulled that as well